shared-doc/build-and-deploy-the-quickstart.adoc

```diff
@@ -70,13 +70,22 @@ endif::mavenCommand[]
 . Make sure you xref:start_the_eap_standalone_server[start the {productName} server] as described above.
 . Open a terminal and navigate to the root directory of this quickstart.
 ifdef::reactive-messaging[]
-. Run this command to enable the MicroProfile Reactive Messaging functionality on the server
+. Run this command to enable the MicroProfile Reactive Messaging functionality on the server:
```

Comment:
Please revert this change, I will actually move this content to the mp reactive README.adoc elsewhere